lib/dbservice/groups/group.ex

```diff
@@ -4,18 +4,56 @@ defmodule Dbservice.Groups.Group do
   use Ecto.Schema
   import Ecto.Changeset
 
+  alias Dbservice.Users.Student
+  alias Dbservice.Groups.GroupStudent
+  alias Dbservice.Users.User
+  alias Dbservice.Sessions.Session
+
   schema "group" do
-    field :input_schema, :map
-    field :locale, :string
-    field :locale_data, :map
+    field :name, :string
+    field :parent_id, :integer
+    field :type, :string
+    field :program_type, :string
+    field :program_sub_type, :string
+    field :program_mode, :string
+    field :program_start_date, :date
+    field :program_target_outreach, :integer
+    field :program_product_used, :string
+    field :program_donor, :string
+    field :program_state, :string
+    field :batch_contact_hours_per_week, :integer
+    field :group_input_schema, :map
+    field :group_locale, :string
+    field :group_locale_data, :map
+
+    has_many :student, Student
+    has_many :group_student, GroupStudent
```

Comment:
let me check a bit on this. I think we can skip the connection between a group and a student and rather use group user connection only. We can anyway find the student related to the user, which in turn would be related to the group.

Let's remove group_student altogether